### PR TITLE
Closures in Connection prevent serialization

### PR DESCRIPTION
**Pull request by FGM**

### CODE REVIEW

**Comment by fubhy** [comment]
This is also required for the Drupal integration as we are now starting to add Relay compliance.

File: src/Relay/Connection/Connection.php [hunk 1]
```diff
@@ -135,4 +129,16 @@ public static function getPageInfoType()
         ]);
     }
 
+    public static function getEdges($value)
+    {
+      return isset($value['edges']) ? $value['edges'] : null;
```
Comment (fubhy) [inline src/Relay/Connection/Connection.php]: Indentation should be 4 spaces.

File: src/Relay/Connection/Connection.php [hunk 2]
```diff
@@ -135,4 +129,16 @@ public static function getPageInfoType()
...
+    }
+
+    public static function getPageInfo($value) {
+      return isset($value['pageInfo']) ? $value['pageInfo'] : null;
```
Comment (fubhy) [inline src/Relay/Connection/Connection.php]: Indentation should be 4 spaces. Also the opening braces should be on a new line.

File: src/Relay/Connection/Connection.php [hunk 3]
```diff
@@ -135,4 +129,16 @@ public static function getPageInfoType()
...
+    }
+
+    public static function getNode($value) {
+      return isset($value['node']) ? $value['node'] : null;
```
Comment (fubhy) [inline src/Relay/Connection/Connection.php]: Indentation should be 4 spaces. Also the opening braces should be on a new line.

File: src/Relay/Connection/Connection.php (hunk 1 above)
Comment (FGM) [inline src/Relay/Connection/Connection.php]: Fixed

File: src/Relay/Connection/Connection.php (hunk 2 above)
Comment (FGM) [inline src/Relay/Connection/Connection.php]: Fixed

File: src/Relay/Connection/Connection.php (hunk 3 above)
Comment (FGM) [inline src/Relay/Connection/Connection.php]: Fixed

**Pull request merged**